### PR TITLE
Add Exercise 6

### PR DESCRIPTION
**Pull request by nguyenvanman**

### CODE REVIEW

File: Exercise_6/Program.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            Console.ReadKey();
+        }
+
+        static void ConvertXmlToJson(string xmlFilePath, string jsonFilePath)
+        {
+            XmlDocument xmlFile = new XmlDocument();
+            xmlFile.LoadXml(ReadFromFile(xmlFilePath));
+            WriteToFile(JsonConvert.SerializeXmlNode(xmlFile), jsonFilePath);
+        }
+        static string ReadFromFile(string filePath)
```
Comment (voquanghoa) [inline Exercise_6/Program.cs]: Read comments from other pull requests.

Comment (nguyenvanman) [reply Exercise_6/Program.cs]: Done

File: Exercise_6/Program.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        }
+
+        static void ConvertXmlToJson(string xmlFilePath, string jsonFilePath)
+        {
+            XmlDocument xmlFile = new XmlDocument();
+            xmlFile.LoadXml(ReadFile(xmlFilePath));
+            WriteFile(jsonFilePath, JsonConvert.SerializeXmlNode(xmlFile));
+        }
+    
+        static string ReadFile(string filePath)
```
Comment (voquanghoa) [inline Exercise_6/Program.cs]: Remove this function.

File: Exercise_6/Program.cs [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                return File.ReadAllText(filePath);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e.Message);
+                throw;
+            }
+        }
+
+        static void WriteFile(string filePath, string content)
```
Comment (voquanghoa) [inline Exercise_6/Program.cs]: Remove this function.

File: Exercise_6/Program.cs (hunk 2 above)
Comment (nguyenvanman) [reply Exercise_6/Program.cs]: Done

File: Exercise_6/Program.cs (hunk 3 above)
Comment (nguyenvanman) [reply Exercise_6/Program.cs]: Done

**Pull request merged**